docs/benchmark.md

```diff
@@ -0,0 +1,108 @@
+# Benchmark baseline results
+
+This document contains baseline benchmark results for Observatorium API Gateway under synthetic load.
+
+Tested on:
+
+```txt
+(15-inch, 2019)
+2,6 GHz 6-Core Intel Core i7
+16 GB 2400 MHz DDR4
+```
+
+Generated using:
+
+[embedmd]:# (../tmp/load_help.txt)
+```txt
+load.sh [-h] [-r n] [-c n] [-m n] [-q n] [-o csv|gnuplot] -- program to test synthetic load on observatorium gateway and report results.
+
+where:
+    -h  show this help text
+    -r  set number of seconds to run (default: 300)
+    -c  set number of cluster to simulate (default: 5000)
+    -m  set number of machines per cluster to simulate (default: 2)
+    -q  set number of concurrent queries to execute (default: 10)
+    -o  set the output format (default: csv. options: csv, gnuplot)
+```
+
+With parameters:
+
+```console
+$ ./test/load.sh -r 300 -c 1000 -m 3 -q 10 -o gnuplot
+```
+
+> It runs tets for 5 minutes, simulating 3000 machines sending metrics and 10 consumers querying for their data every second.
+> Observatorim API GW runs in-front of 1 Thanos Receive and 1 Thanos Querier.
+
+## Results
```

Comment:
shouldn't you also include some formula how to calculate the overall resource requremenent for a given load.
I mean something that will answer the  question: what resources would I need for  ingesting 1000 metrics and querying .... 

I remember that Clayton also wanted to know the total amout of memory needed when running a given query  so probably a good idea to add something like this as a follow up PR

Reply:
I will add a section to document to explain this.